src/util/hass-translation.ts

```diff
@@ -1,42 +1,43 @@
 import { translationMetadata } from "../resources/translations-metadata";
+import { fetchFrontendUserData } from "../data/frontend-user-data";
+import { HomeAssistant } from "../types";
 
-export function getActiveTranslation() {
-  // Perform case-insenstive comparison since browser isn't required to
-  // report languages with specific cases.
-  const lookup = {};
-  /* eslint-disable no-undef */
-  Object.keys(translationMetadata.translations).forEach((tr) => {
-    lookup[tr.toLowerCase()] = tr;
-  });
+// Perform case-insenstive comparison since browser isn't required to
+// report languages with specific cases.
+const TRANSLATION_LOOKUP = {};
```

Comment:
We should not do this in the module code, it will slow down the loading of Home Assistant. Keep it in the function so that we execute it when we need it.

Reply:
Done